taxcalc/parameters.py

```diff
@@ -6,7 +6,7 @@
 
 DEFAULT_START_YEAR = 2013
 
-class Parameters(object):
+class Params(object):
```

Comment:
I think it's generally expected that object names are never abbreviated, but parameter names (including those that refer to objects) can be depending on context. Even in Ruby, `parameters` is one of those words used so commonly that we're ok abbreviating it to `params`. So I would doubly expect python coders to understand:
```
paramsX = Parameters(start_year=2013)
calcX = Calculator(records=recordsX, params=paramsX)
```
pretty easily.